docs/docs-content/integrations/harbor-edge.md

```diff
@@ -209,6 +209,80 @@ docker push 10.10.137.220:30003/spectro-images/alpine:latest
   </TabItem>
   </Tabs>
 
+#### Enable Image Download from Outside of Harbor
+
+When a cluster uses the Harbor Edge-Native Config pack, the cluster assumes all images will be stored in Harbor after
+the initial download. While it's true that all images in the cluster profile for that cluster and managed by Palette
+will be stored in the Harbor registry, images that are not managed by Palette will not exist in the Harbor registry.
```

Comment:
@lennessyy The first three sentences were a bit challenging to read. I took a stab at simplifying the message, take a peek and see what you think.

> If a cluster is configured with the Harbor Edge-Native Config pack, it will assume that all images will be stored in Harbor once they are initially downloaded. However, it is important to note that only the images that are part of the cluster profile and managed by Palette will be stored in the Harbor registry. Any images not managed by Palette will not be available in the Harbor registry.